src/fal/cli/flow_runner.py

```diff
@@ -63,8 +64,9 @@ def run_threaded(
     filtered_graph = FilteredGraph.from_execution_plan(
         origin_graph, execution_plan=execution_plan
     )
-    planned_graph = PlannedGraph.from_filtered_graph(
-        filtered_graph, enable_chunking=False
+    shuffled_graph = ShuffledGraph.from_filtered_graph(filtered_graph)
+    planned_graph = PlannedGraph.from_shuffled_graph(
+        shuffled_graph, enable_chunking=False
```

Comment:
ATL: What's happening here?

Reply:
`PlannedGraph.from_shuffled_graph` does nothing other than wrap the graph when `enable_chunking=False`